### ISSUE TITLE
Installing package on docker causes "Server did not respond to request for update" error

### ISSUE THREAD

**Issue by cantsin** [jhpyle/docassemble#360]
:wave: When going through the "hello world" [tutorial](https://docassemble.org/docs/helloworld.html) everything was working nicely until I got to the stage where I wanted to `Install` a package. Clicking said button takes me to a "Updating packages..." waiting page. This page eventually times out with an error: "Server did not respond to request for update". This is with the `helloworld` example copied and pasted verbatim.

I am running a docker image straight up -- `docker run -p 8888:80 -p 443:443 --stop-timeout 600 jhpyle/docassemble` -- should I be increasing the timeout? Something else?

The package seems to be successfully installed and is available as an interview, so the server shouldn't be timing out.

**Comment by jhpyle** [comment]
What version are you running?

**Comment by cantsin** [comment]
Looks like I'm running `1.2.28` -- will update to the latest and try again.

**Comment by cantsin** [comment]
(Apologies, I did not realize docassemble was updated frequently, I should have checked in the first place :man_facepalming:). 

`docker run -it jhpyle/docassemble cat /usr/share/docassemble/webapp/VERSION` gives me `1.2.30`

Now I get a non-error message: `The package update did not report an error. The logs are below.` The top line says, `docassemble.interview: pip install command returned success code.  See log for details.`

Even though this message is confusing, I _think_ this is expected? Thank you -- I'll close -- apologies for the noise!
